library/models/library.py

```diff
@@ -205,24 +255,22 @@ def _check_issue_book_limit(self):
                     else:
                         # Check the book issue limit on card if it is over it
                         # give warning
-                        raise UserError(_('Book issue limit is over on this\
-                        card'))
+                        raise ValidationError(_('''Book issue limit is over
+                                                on this  card'''))
```

Comment:
correct spacing between 'this' and 'card'. add exclamation.

Reply:
done